components/admin_console/team_channel_settings/channel/details/channel_details.tsx

```diff
@@ -440,7 +444,7 @@ export default class ChannelDetails extends React.PureComponent<ChannelDetailsPr
             filter((g) => {
                 return origGroups.some((group) => group.id === g.id && group.scheme_admin !== g.scheme_admin);
             }).
-            map((g) => actions.patchGroupSyncable(g.id, channelID, Groups.SYNCABLE_TYPE_CHANNEL, {scheme_admin: g.scheme_admin}));
+            map((g) => actions.patchGroupSyncable(g.id, channelID, Groups.SYNCABLE_TYPE_CHANNEL, {scheme_admin: g.scheme_admin, auto_add: false}));
```

Comment:
How come we're adding this extra field here?

Reply:
`TS2345: Argument of type '{ scheme_admin: boolean; }' is not assignable to parameter of type 'SyncablePatch'.   Property 'auto_add' is missing in type '{ scheme_admin: boolean; }' but required in type 'SyncablePatch'.`

Reply:
Can we change the method signature back to `Partial<SyncablePatch>` to allow this field to be removed?
The reason is the field is patched all the way up to the server and I'm not sure if it checks for `false` or `null` there, since it's represented by a pointer which could be null.

That or we may need to do another redux update.

Reply:
ok, fixed in f76b723